mesheryctl/cmd/start.go

```diff
@@ -42,33 +45,75 @@ var startCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		fmt.Println("Starting Meshery...")
+		log.Info("Starting Meshery...")
 		start := exec.Command("docker-compose", "-f", dockerComposeFile, "up", "-d")
 		start.Stdout = &out
 		start.Stderr = &stderr
 
 		if err := start.Run(); err != nil {
-			fmt.Println(stderr.String())
+			log.Println(stderr.String())
 			return
 		}
 
-		fmt.Println("Opening Meshery in your broswer. If Meshery does not open, please point your browser to http://localhost:9081 to access Meshery.")
-		ostype, err := exec.Command("uname", "-s").Output()
+		checkFlag := 0 //flag to check
+
+		//connection to docker-client
+		cli, err := client.NewEnvClient()
 		if err != nil {
-			log.Fatal("[WARNING] Unable to detect OS type. Warning message: \n", err)
+			panic(err)
```

Comment:
May be we should follow a pattern, either use panic or log.Fatal? Lets stick with log.Fatal